### PR TITLE
Ux/Design

### PR DESCRIPTION
**Pull request by yohanology**
The design update.

### CODE REVIEW

**Comment by h8rry** [comment]
You have **69,738 additions**

**Comment by h8rry** [comment]
You shouldn't need to run `rake asset:precompile`. If you do need to run it, you should remove the files.

**Comment by h8rry** [comment]
Heroku will automatically precompile assets

**Comment by yohanology** [comment]
Deleted.

File: app/assets/javascripts/angular/controller/forms/updateForms.js.erb [hunk 1]
```diff
@@ -18,21 +18,29 @@ app.filter('orderFieldBy', function(){
 
 app.controller('FormsUpdateCtrl', ['$scope', '$http', '$auth', '$location', '$routeParams','$rootScope','dragulaService', '$timeout', function($scope, $http, $auth, $location, $routeParams, $rootScope, dragulaService, $timeout){
 
-  $rootScope.title = 'Design';
+  $rootScope.title = 'Build';
```
Comment (h8rry) [inline app/assets/javascripts/angular/controller/forms/updateForms.js.erb:21]: Why do you need to use rootScope?

**Comment by h8rry** [comment]
@yohanology please review

**Pull request merged**